tests/bind_module_class.cpp

```diff
@@ -0,0 +1,17 @@
+#include <mruby.hpp>
+
+class Cat
+{
+public:
+	Cat(int a)
+	{}
+};
+
+int main()
+{
+	mruby::VM vm;
+	auto module = vm.create_module("Animals");
+	module->create_class<Cat, int>("Cat");
+    auto cls = module->get_class("Cat");
+	return cls.get() ? 0 : 1;
```

Comment:
I think like the others this should also throw an exception

Reply:
Actually never mind. Was half asleep. This is ok. I thought it checked that it would throw like the rest.